frameworks/PHP/ubiquity/ubiquity-swoole.dockerfile

```diff
@@ -23,7 +19,7 @@ RUN deploy/run/install-composer.sh
 RUN apt-get update -yqq > /dev/null && \
     apt-get install -yqq git unzip > /dev/null
 
-RUN php composer.phar require phpmv/ubiquity-devtools:dev-master phpmv/ubiquity-swoole:dev-master --quiet
+RUN php composer.phar require phpmv/ubiquity-devtools:dev-techempower-benchmarks phpmv/ubiquity-swoole:dev-techempower-benchmarks --quiet
```

Comment:
I don't think, that have a branch only for the benchmark, is considered a production grade.
I understand if it is to do a test.

Reply:
It's just like a security airlock.
It allows me to prepare the next bench (on the `dev-techempower` branch) without taking any risk on `dev-master`.
There are times when the opposite is also likely: I make modifications on `master` that make Techempower tests no longer pass...
Once the test is passed, if it is a success, I make a merge between master and techempower branchs.
The Techempower branch is not intended to be very far from the master branch...

Once the test is passed, if it is a success, I make a merge between master and techempower